README.md

```diff
@@ -25,3 +27,39 @@ JdbcStorageFactory.newBuilder()
                   .setDataSource(dataSource)
                   .build();
 ```
+
+### Type mapping
```

Comment:
`SQL Type Mapping`.

Also, please make this one a main entry. And make the next two as sub-entries. Let's discuss if needed.

Reply:
Done.